src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -140,12 +140,13 @@ void RaiseSecond()
             VisualStudio.Editor.Verify.TextContains("Second?.");
         }
 
-        [CriticalWpfFact]
+        [CriticalWpfTheory]
         [Trait(Traits.Feature, Traits.Features.EditorConfig)]
         [Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)]
         [WorkItem(15003, "https://github.com/dotnet/roslyn/issues/15003")]
         [WorkItem(19089, "https://github.com/dotnet/roslyn/issues/19089")]
-        public void ApplyEditorConfigAndFixAllOccurrences()
+        [IterationData(20)]
+        public void ApplyEditorConfigAndFixAllOccurrences(int iteration)
```

Comment:
This is obviously not going to be merged, but want plenty of runs for confidence.

Reply:
You forgot to assign it to a discard so it failed the build 😆 

```csharp
_ = iteration;
```

Reply:
This is why warning-as-error is great to have locally too...

Reply:
This will give you the same on the command line:

> .\Build.cmd -rebuild